packages/victory-core/src/victory-util/wrapper.js

```diff
@@ -204,6 +208,42 @@ export default {
     const { polar, startAngle, endAngle, categories, minDomain, maxDomain } = props;
     const parentProps = { polar, startAngle, endAngle, categories, minDomain, maxDomain };
     let stack = 0;
+    const children = childComponents
+      ? childComponents.slice(0)
+      : React.Children.toArray(props.children);
+
+    const childrenAreHistograms = children.some((child) => {
```

Comment:
is this check sufficient?

Reply:
Since stacking histograms will be incompatible with other types, I think we need to check on  whether `.some`children are histograms,  and then warn if not `.every` child is a histogram.